gobblin-compaction/src/main/java/gobblin/compaction/hivebasedconstructs/MRCompactionEntity.java

```diff
@@ -0,0 +1,28 @@
+package gobblin.compaction.hivebasedconstructs;
+
+import java.util.Properties;
+import lombok.Getter;
+
+
+/**
+ * Entity that stores information required for launching an {@link gobblin.compaction.mapreduce.MRCompactor} job
+ *
+ * {@link #primaryKeyInfo}: Comma delimited list of fields to use as primary key
+ * {@link #deltaInfo}: Comma delimited list of fields to use as deltaInfo
+ * {@link #location}: Location of files associated with table
+ * {@link #props}: Other properties to be passed to {@link gobblin.compaction.mapreduce.MRCompactor}
+ */
+@Getter
+public class MRCompactionEntity {
+  private final String primaryKeyInfo;
```

Comment:
Further, a name like primaryKeysList is more descriptive then primaryKeyInfo. Same for dataFilesPath vs location.